.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,96 @@
+name: Tests and lint
```

Comment:
I've usually seen the workflow filename align with the name of the workflow. It doesn't affect anything, just FYI.

Reply:
I can definitely rename it 👍